src/test/java/org/apache/accumulo/access/AccessExpressionBenchmark.java

```diff
@@ -117,8 +117,8 @@ List<String> getStringExpressions() {
       return allTestExpressionsStr;
     }
 
-    public ArrayList<EvaluatorTests> getEvaluatorTests() {
-      return evaluatorTests;
+    public List<EvaluatorTests> getEvaluatorTests() {
+      return List.copyOf(evaluatorTests);
```

Comment:
This copy will be done on each run of the benchmark that call this function, so it would be good to avoid it.

Reply:
I switched to Collections.unmodifiableList instead. That avoids the copy, but keeps spotbugs happy because the encapsulated field is still only exposed in an immutable way.